CHANGELOG-3.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since v2.2.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Update volumeAttachment to v1.
+- RBAC policy was updated to allow the external-attacher to patch VolumeAttachment.Status ([#200](https://github.com/kubernetes-csi/external-attacher/pull/200), [@cwdsuzhou](https://github.com/cwdsuzhou))
+
+## Changes by Kind
+
+### Feature
```

Comment:
Do you want to include a node about vsphere csi migration?

Reply:
Added